app/views/collab_mailer/invite_to_collaborate.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Hey there <%= @to_user.username %></p>
+<p><%= @from_user.username %> would like to collaborate with you on <%= @skills.join(', ').sub(/.*\K,\s/, ' or ') %>.</p>
+<p>Reach out to him via email at <%= mail_to @from_user.email, @from_user.email %> or via Slack.</p>
```

Comment:
why two @from_user.email?

Reply:
Also, could there not be two links for accepting or decline within the email which replies for you rather than having to email directly?

Reply:
1. you need to pass both arguments (one for text, the other for the actual link) otherwise it breaks in the mailer
2. we can add that in a later iteration